package.json

```diff
@@ -10,7 +10,7 @@
   "license": "Apache-2.0",
   "bugs": "https://github.com/redhat-developer/vscode-quarkus/issues",
   "engines": {
-    "vscode": "^1.37.0"
+    "vscode": "^1.65.0"
```

Comment:
most likely will be incompatible with theia

Reply:
@benoitf what's the max vscode engine version compatible with theia?

Reply:
Yeah, it's not supported currently. https://eclipse-theia.github.io/vscode-theia-comparator/status.html

Also, from https://github.com/redhat-developer/vscode-java/pull/2183#pullrequestreview-790383636, looks like updating the engine itself should not be harmful to Theia. Just have to ensure you don't call API that doesn't exist.

Reply:
Also see https://github.com/redhat-developer/vscode-java/pull/1975/files for how to adopt new API while still being compatible with older engines.